android/modules/ui/src/java/ti/modules/titanium/ui/WebViewProxy.java

```diff
@@ -37,7 +37,8 @@
 	TiC.PROPERTY_WEBVIEW_IGNORE_SSL_ERROR,
 	TiC.PROPERTY_OVER_SCROLL_MODE,
 	TiC.PROPERTY_CACHE_MODE,
-	TiC.PROPERTY_LIGHT_TOUCH_ENABLED
+	TiC.PROPERTY_LIGHT_TOUCH_ENABLED,
+	TiC.PROPERTY_CONTEXT_MENU_ITEMS
```

Comment:
I think this ticket is just about addressing the ability to disable the context menu on text selection. So we shouldn't need a new contextMenuItems property. But we should need a new disableContextMenu property.